kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -65,9 +67,10 @@
 
 class DerivedResourcesTest {
 
-    static final YAMLMapper YAML_MAPPER = new YAMLMapper()
+    static final ObjectMapper YAML_MAPPER = new YAMLMapper()
```

Comment:
Should we configure our own mapper here or should we use the kubernetes-client one. There are pros and cons each way I guess